### ISSUE TITLE
Missing links in docs

### ISSUE THREAD

**Issue by jni** [napari/napari#2474]
## 📚 Documentation
While investigating napari/napari#2473 I also found these lines in the [build](https://github.com/napari/napari/runs/2195089768?check_suite_focus=true#step:6:2430):

```
/home/runner/work/napari/napari/main/docs/community/governance.md:45: WARNING: 'myst' reference target not found: CONTRIBUTING.md
/home/runner/work/napari/napari/main/docs/community/governance.md:49: WARNING: 'myst' reference target not found: CORE_DEV_GUIDE.md
/home/runner/work/napari/napari/main/docs/community/governance.md:88: WARNING: 'myst' reference target not found: MISSION_AND_VALUES.md
/home/runner/work/napari/napari/main/docs/community/governance.md:108: WARNING: 'myst' reference target not found: MISSION_AND_VALUES.md
WARNING: Found tableofcontents directive in community/index, but this file has no descendents.
/home/runner/work/napari/napari/main/docs/community/mission_and_values.md:3: WARNING: 'myst' reference target not found: CORE_DEV_GUIDE.md
/home/runner/work/napari/napari/main/docs/community/mission_and_values.md:3: WARNING: 'myst' reference target not found: GOVERNANCE
/home/runner/work/napari/napari/main/docs/community/mission_and_values.md:43: WARNING: 'myst' reference target not found: CORE_DEV_GUIDE.md
/home/runner/work/napari/napari/main/docs/community/mission_and_values.md:43: WARNING: 'myst' reference target not found: CODE_OF_CONDUCT.md
/home/runner/work/napari/napari/main/docs/developers/contributing.md:144: WARNING: 'myst' reference target not found: TESTING.md
/home/runner/work/napari/napari/main/docs/developers/contributing.md:201: WARNING: 'myst' reference target not found: CODE_OF_CONDUCT.md
/home/runner/work/napari/napari/main/docs/developers/core_dev_guide.md:12: WARNING: 'myst' reference target not found: MISSION_AND_VALUES.md
/home/runner/work/napari/napari/main/docs/developers/core_dev_guide.md:33: WARNING: 'myst' reference target not found: CODE_OF_CONDUCT.md
/home/runner/work/napari/napari/main/docs/developers/core_dev_guide.md:46: WARNING: 'myst' reference target not found: CONTRIBUTING.md
/home/runner/work/napari/napari/main/docs/developers/core_dev_guide.md:72: WARNING: 'myst' reference target not found: MISSION_AND_VALUES.md
/home/runner/work/napari/napari/main/docs/developers/core_dev_guide.md:79: WARNING: 'myst' reference target not found: BENCHMARKS.md
/home/runner/work/napari/napari/main/docs/developers/core_dev_guide.md:146: WARNING: 'myst' reference target not found: CONTRIBUTING.md
/home/runner/work/napari/napari/main/docs/developers/core_dev_guide.md:147: WARNING: 'myst' reference target not found: CODE_OF_CONDUCT.md
/home/runner/work/napari/napari/main/docs/developers/core_dev_guide.md:148: WARNING: 'myst' reference target not found: GOVERNANCE.md
/home/runner/work/napari/napari/main/docs/developers/core_dev_guide.md:149: WARNING: 'myst' reference target not found: MISSION_AND_VALUES.md
/home/runner/work/napari/napari/main/docs/developers/core_dev_guide.md:150: WARNING: 'myst' reference target not found: BENCHMARKS.md
/home/runner/work/napari/napari/main/docs/community/governance.md:45: WARNING: 'myst' reference target not found: CONTRIBUTING.md
/home/runner/work/napari/napari/main/docs/community/governance.md:49: WARNING: 'myst' reference target not found: CORE_DEV_GUIDE.md
/home/runner/work/napari/napari/main/docs/community/governance.md:88: WARNING: 'myst' reference target not found: MISSION_AND_VALUES.md
/home/runner/work/napari/napari/main/docs/community/governance.md:108: WARNING: 'myst' reference target not found: MISSION_AND_VALUES.md
/home/runner/work/napari/napari/main/docs/community/mission_and_values.md:3: WARNING: 'myst' reference target not found: CORE_DEV_GUIDE.md
/home/runner/work/napari/napari/main/docs/community/mission_and_values.md:3: WARNING: 'myst' reference target not found: GOVERNANCE
/home/runner/work/napari/napari/main/docs/community/mission_and_values.md:43: WARNING: 'myst' reference target not found: CORE_DEV_GUIDE.md
/home/runner/work/napari/napari/main/docs/community/mission_and_values.md:43: WARNING: 'myst' reference target not found: CODE_OF_CONDUCT.md
/home/runner/work/napari/napari/main/docs/roadmaps/0_3.md:75: WARNING: 'myst' reference target not found: MISSION_AND_VALUES.md
/home/runner/work/napari/napari/main/docs/roadmaps/0_3.md:75: WARNING: 'myst' reference target not found: GOVERNANCE.md
/home/runner/work/napari/napari/main/docs/roadmaps/0_3.md:75: WARNING: 'myst' reference target not found: CONTRIBUTING.md
/home/runner/work/napari/napari/main/docs/roadmaps/0_3_retrospective.md:21: WARNING: 'myst' reference target not found: ROADMAP_0_4.md
/home/runner/work/napari/napari/main/docs/roadmaps/0_3_retrospective.md:115: WARNING: 'myst' reference target not found: MISSION_AND_VALUES.md
/home/runner/work/napari/napari/main/docs/roadmaps/0_3_retrospective.md:115: WARNING: 'myst' reference target not found: GOVERNANCE.md
/home/runner/work/napari/napari/main/docs/roadmaps/0_3_retrospective.md:115: WARNING: 'myst' reference target not found: CONTRIBUTING.md
/home/runner/work/napari/napari/main/docs/roadmaps/0_4.md:134: WARNING: 'myst' reference target not found: MISSION_AND_VALUES.md
/home/runner/work/napari/napari/main/docs/roadmaps/0_4.md:134: WARNING: 'myst' reference target not found: GOVERNANCE.md
/home/runner/work/napari/napari/main/docs/roadmaps/0_4.md:134: WARNING: 'myst' reference target not found: CONTRIBUTING.md
/home/runner/work/napari/napari/main/docs/roadmaps/0_3.md:75: WARNING: 'myst' reference target not found: MISSION_AND_VALUES.md
/home/runner/work/napari/napari/main/docs/roadmaps/0_3.md:75: WARNING: 'myst' reference target not found: GOVERNANCE.md
/home/runner/work/napari/napari/main/docs/roadmaps/0_3.md:75: WARNING: 'myst' reference target not found: CONTRIBUTING.md
/home/runner/work/napari/napari/main/docs/roadmaps/0_3_retrospective.md:21: WARNING: 'myst' reference target not found: ROADMAP_0_4.md
/home/runner/work/napari/napari/main/docs/roadmaps/0_3_retrospective.md:115: WARNING: 'myst' reference target not found: MISSION_AND_VALUES.md
/home/runner/work/napari/napari/main/docs/roadmaps/0_3_retrospective.md:115: WARNING: 'myst' reference target not found: GOVERNANCE.md
/home/runner/work/napari/napari/main/docs/roadmaps/0_3_retrospective.md:115: WARNING: 'myst' reference target not found: CONTRIBUTING.md
/home/runner/work/napari/napari/main/docs/roadmaps/0_4.md:134: WARNING: 'myst' reference target not found: MISSION_AND_VALUES.md
/home/runner/work/napari/napari/main/docs/roadmaps/0_4.md:134: WARNING: 'myst' reference target not found: GOVERNANCE.md
/home/runner/work/napari/napari/main/docs/roadmaps/0_4.md:134: WARNING: 'myst' reference target not found: CONTRIBUTING.md
WARNING: Found tableofcontents directive in roadmaps/index, but this file has no descendents.
```

Sure enough, those seem to correspond to missing links in our napari.org website.

**Comment by sofroniewn** [comment]
Thanks for catching @jni, maybe @kne42 can look into when the week begins. It might be nice to use a tool to check for missing links. I think @GenevieveBuckley had proposed one on an earlier issue, but i can't find it now

**Comment by GenevieveBuckley** [comment]
I think it was this one: https://github.com/linkchecker/linkchecker

I haven't used it, but it seems like it fits the job. Discussed here in https://github.com/napari/docs/issues/439

**Comment by jni** [comment]
Linkchecker won't work here because Jupyter Book simply does not include the hyperlink in the html if the target is missing in the sphinx source file (MyST). So we have to look at the build logs.

**Comment by GenevieveBuckley** [comment]
Would this work for MyST source files? https://github.com/marketplace/actions/markdown-link-check

**Comment by jni** [comment]
Probably not. They are sphinx references which work differently from pure markdown. Though that's very cool.

**Comment by sofroniewn** [comment]
This has now been fixed by napari/napari#2562 and other PRs
